src/lib/plot/axis/axis-ticks.js

```diff
@@ -51,16 +51,10 @@ function _getTickFormatFn(scale, tickTotal, tickFormat) {
     tickFormat;
 }
 
-function _getTickValues(scale, tickTotal, tickValues) {
-  return !tickValues ?
-    (scale.ticks ? scale.ticks(tickTotal) : scale.domain()) :
-    tickValues;
-}
-
 function AxisTicks(props) {
   const {
     attr,
-    orientation,
+    position,
```

Comment:
@chrisirhc , agree on that, but we have an ambiguity between `vertical`|`horizontal` orientations and `top`|`left`|`right`|`bottom` orientations as well.

I doubt that it makes any sense to reuse the sides for the grid, thus I decided to rename `orientation` to `position`.

One more thing: in 0.4.x the `orientation` of Axis is overriden by the XAxis/YAxis, so there should be no confusion